mlflow/tracking/client.py

```diff
@@ -380,37 +380,6 @@ def create_run(
         """
         return self._tracking_client.create_run(experiment_id, start_time, tags, run_name)
 
-    def _create_trace_info(
```

Comment:
Shall we define private `_start_trace()` and `_end_trace()` methods that the trace collector can use?

Reply:
Yup planning to do that in the next PR to actually using it in trace creation logic.